### ISSUE TITLE
PHP critical error on webapp install

### ISSUE THREAD

**Issue by fabgg** [symfony/symfony#58841]
### Symfony version(s) affected

6.4.13

### Description

after installation process of fresh install with ```composer require webapp``` the cache clear commande generate a critical php error 
```shell
Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 255
!!  
!!   // Clearing the cache for the dev environment with debug true                  
!!  
!!  15:33:47 CRITICAL  [php] Uncaught Error: Too few arguments to function PHPStan\PhpDocParser\Parser\ConstExprParser::__construct(), 0 passed in /var/www/test/sf64/vendor/symfony/property-info/Extractor/PhpStanExtractor.php on line 71 and exactly 1 expected ["exception" => ArgumentCountError { …}]
!!  
!!  In ConstExprParser.php line 18:
!!                                                                                 
!!    Too few arguments to function PHPStan\PhpDocParser\Parser\ConstExprParser::  
!!    __construct(), 0 passed in /var/www/test/sf64/vendor/symfony/property-info/  
!!    Extractor/PhpStanExtractor.php on line 71 and exactly 1 expected             
!!                                                                                 
!!  
!!  cache:clear [--no-warmup] [--no-optional-warmers]
!!  
!!  
Script @auto-scripts was called via post-update-cmd
```

the php version is 
```shell
PHP 8.2.24 (cli) (built: Sep 27 2024 04:16:10) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.24, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.24, Copyright (c), by Zend Technologies
```

I got same result on local docker based on bookworm php 8.2.25
```
PHP 8.2.25 (cli) (built: Nov 12 2024 07:25:23) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.25, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.25, Copyright (c), by Zend Technologies
```

### How to reproduce

This error still append with symfony installer
```shell
symfony new test_app --version="6.4.*" --webapp
```

**Comment by stof** [comment]
Duplicate of https://github.com/symfony/symfony/issues/58796

**Comment by stof** [comment]
This is already fixed in the dev version thanks to #58800 and #58804. The fix will be part of the next patch release.

**Comment by endelwar** [comment]
pinning `phpdocumentor/reflection-docblock` to `5.5.1` fixed the issues for me, waiting for the next path release :)
